src/squirrel/internal/query.gleam

```diff
@@ -65,6 +65,39 @@ pub fn add_types(
 ) -> Result(TypedQuery, Error) {
   let UntypedQuery(file:, name:, comment:, content:, starting_line:) = query
 
+  // Quick and dirty hack to allow nullable parameters in insert queries
+  // Looks for a comment like this: '-- nullable: $1, $3' in the sql file
+  // and makes the specified parameters nullable
```

Comment:
Could you please remove this code from the PR? I like the idea and definitely want to add this in the future, but let's keep this PR focused on just the fix for now :)

Reply:
Oops, that wasn't supposed to be there 😅